### PR TITLE
Fix Android USAGE.md import process

### LINKED ISSUE #613: Adopt openhab-android repo name
e.g. in update-external-resources script.

https://github.com/openhab/openhab-android/pull/588

### LINKED ISSUE #617: Images on apps/android.html are not shown
See http://docs.openhab.org/addons/uis/apps/android.html

### PR DESCRIPTION
**Pull request by ThomDietrich**
Fixes #613 
Fixes #617 

CC @mueller-ma 

Everyone: I'm going to merge directly in order to get on with another task. Please carry out your review as usual, I can open another PR with whatever comes up!

Signed-off-by: Thomas Dietrich <thomas.dietrich@tu-ilmenau.de> (github: ThomDietrich)

### CODE REVIEW

File: addons/uis/apps/android.md [hunk 1]
```diff
@@ -29,14 +33,17 @@ The openHAB Android application is a native client for openHAB. The app follows
 ## Getting Started
 
 When first installed the app is in "Demo Mode".
-To connect it to your own openHAB server, first navigate to Settings and uncheck the "Demo Mode" option.
-Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to autodetect to your openHAB server if it is on the same network.
+To connect to your own openHAB server, first navigate into the Settings menu and uncheck the "Demo Mode" option.
```
Comment (bgilmer77) [inline addons/uis/apps/android.md]: Consider, "navigate to the..."

File: addons/uis/apps/android.md [hunk 2]
```diff
@@ -29,14 +33,17 @@ The openHAB Android application is a native client for openHAB. The app follows
...
 
 You also have the option to manually set the server URL in the settings.
 
-**Local server settings:** Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
+**Local server settings:**
+Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
 
-**Remote server settings:** If you want to use openHAB cloud, please enter URL, user and password of you cloud instance here. Please make sure to [secure this connection](http://docs.openhab.org/installation/security.html).
+**Remote server settings:**
+If you want to use openHAB cloud, please enter URL, user and password of you cloud instance here.
```
Comment (bgilmer77) [inline addons/uis/apps/android.md]: Consider, "If you want to use openHAB cloud, please enter the URL, username and..."

File: addons/uis/apps/android.md [hunk 3]
```diff
@@ -50,11 +57,11 @@ There are a number of strategies available to provide [secure remote access]({{b
 
 ## Permanent Deployment
 
-If you want to use openHAB Android on a wall mounted tablet, go to settings and tick `Disable display timer` and `Full Screen`.
+If you want to use openHAB Android on a wall mounted tablet, go to settings and tick `Disable display timer` and `Fullscreen`.
```
Comment (bgilmer77) [inline addons/uis/apps/android.md]: Consider changing "tick" to "select".

File: addons/uis/apps/ios.md [hunk 4]
```diff
@@ -46,6 +48,6 @@ The openHAB app will henceforth first try connecting to the local URL, then fall
 
 ## Help and Resources
 
-If your question or problem isn't covered here in the documentation pages, check the [openHAB Community Forum](https://community.openhab.org). 
+If your question or problem isn't covered here in the documentation pages, check the [openHAB Community Forum](https://community.openhab.org).
```
Comment (bgilmer77) [inline addons/uis/apps/ios.md]: Consider changing to, "is not covered in these documentation pages..."

File: configuration/multimedia.md [hunk 5]
```diff
@@ -86,7 +86,7 @@ If no voice or no audio sink is provided, the default voice and default audio si
 Although there are already interfaces defined in openHAB for speech-to-text, up to now there is no add-on available for this functionality.
 So the only choice that is available right now is to use the Android voice recognition feature that is built into the openHAB Android app.
 
-The Android app is [not yet adapted to the openHAB 2 voice features](https://github.com/openhab/openhab.android/issues/242).
+The Android app is [not yet adapted to the openHAB 2 voice features](https://github.com/openhab/openhab-android/issues/242).
```
Comment (bgilmer77) [inline configuration/multimedia.md]: Consider changing to, "adapted to openHAB..."

**Comment by ThomDietrich** [comment]
Thanks @Confectrician,
Thanks @bgilmer77. I'll attend to these comments over in the original article: https://github.com/openhab/openhab-android/pull/615

**Pull request merged**